packages/libs/error-stack/src/lib.rs

```diff
@@ -357,19 +400,26 @@
 //!  `hooks`   |Enables the usage of [`set_display_hook`] and [`set_debug_hook`]| `std`   | disabled
 //! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
 //!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
+//! `small`    | Enable optimizations for the memory footprint of [`Report`]    |         | enabled
 //!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
 //!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
 //!
+//! You can enable further size optimizations, by including [`smallvec`] in your dependencies and
+//! enabling the `union` feature. This will reduce the size of [`Report`] by one machine word.
+//!
```

Comment:
Union needs 1.49, so that's fine. I don't think that there's any downside, but I'd still feel kinda iffy to implicitly enable union for every other crate.

Reply:
but maybe I am overthinking this.

Reply:
I think it's fine to simply use the feature 👍🏻